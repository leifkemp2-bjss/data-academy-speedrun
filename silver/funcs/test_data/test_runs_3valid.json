[
  {
    "id": "yoqv235z",
    "weblink": "https://www.speedrun.com/little_nightmares/run/yoqv235z",
    "game": "j1lewj6g",
    "level": "kwjekv19",
    "category": "xd131x4k",
    "comment": "sub 10 is doable",
    "status": {
      "status": "verified",
      "examiner": "j2wqv36j",
      "verify-date": "2024-11-28T20:39:07Z"
    },
    "players": [
      {
        "rel": "user",
        "id": "xk1kw9kj",
        "uri": "https://www.speedrun.com/api/v1/users/xk1kw9kj"
      }
    ],
    "date": "2024-11-27",
    "submitted": "2024-11-27T02:53:54Z",
    "times": {
      "primary": "PT10M1.110S",
      "primary_t": 601.11
    },
    "system": {
      "platform": "8gej2n93",
      "emulated": false,
      "region": null
    },
    "splits": null,
    "values": {}
  },
  {
    "id": "menv862m",
    "weblink": "https://www.speedrun.com/Coffee_Caravan/run/menv862m",
    "game": "j1lnmq76",
    "level": null,
    "category": "xk97mn4d",
    "comment": "First PB on the new patch!",
    "status": {
      "status": "verified",
      "examiner": "8gekpnrj",
      "verify-date": "2024-11-27T02:52:34Z"
    },
    "players": [
      {
        "rel": "user",
        "id": "8gekpnrj",
        "uri": "https://www.speedrun.com/api/v1/users/8gekpnrj"
      }
    ],
    "date": "2024-11-26",
    "submitted": "2024-11-27T02:52:34Z",
    "times": {
      "primary": "PT51M56S",
      "primary_t": 3116
    },
    "system": {
      "platform": "8gej2n93",
      "emulated": false,
      "region": null
    },
    "splits": null,
    "values": {
      "ylpy93jl": "ln80m30l",
      "0nwm5r58": "qj74ndeq",
      "dloogd5l": "le2m8epl"
    }
  },
  {
    "id": "zgj37vjz",
    "weblink": "https://www.speedrun.com/fp2/run/zgj37vjz",
    "game": "m1mnlpjd",
    "level": "rw66ngpw",
    "category": "w205p0od",
    "comment": "I was tired of MastaPowa finding all the new timesaves\n\nHigh FPS still cringe though",
    "status": {
      "status": "verified",
      "examiner": "qxkowgk8",
      "verify-date": "2024-11-27T07:28:19Z"
    },
    "players": [
      {
        "rel": "user",
        "id": "68wq40z8",
        "uri": "https://www.speedrun.com/api/v1/users/68wq40z8"
      }
    ],
    "date": "2024-11-27",
    "submitted": "2024-11-27T02:52:07Z",
    "times": {
      "primary": "PT34.380S",
      "primary_t": 34.38
    },
    "system": {
      "platform": "8gej2n93",
      "emulated": false,
      "region": null
    },
    "splits": null,
    "values": {
      "ylqxx138": "1gngy8ol",
      "gnx2e2g8": "81p567vl"
    }
  }
]