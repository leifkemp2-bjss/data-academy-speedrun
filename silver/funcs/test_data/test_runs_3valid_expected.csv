id,game,level,category,date,submitted,players,reviewer,status,status_comment,time,submitted_date
yoqv235z,j1lewj6g,kwjekv19,xd131x4k,27/11/2024,2024-11-27T02:53:54Z,"[{""id"":""xk1kw9kj"",""name"":null,""rel"":""user""}]",j2wqv36j,verified,null,601.11,27/11/2024
menv862m,j1lnmq76,null,xk97mn4d,26/11/2024,2024-11-27T02:52:34Z,"[{""id"":""8gekpnrj"",""name"":null,""rel"":""user""}]",8gekpnrj,verified,null,3116,27/11/2024
zgj37vjz,m1mnlpjd,rw66ngpw,w205p0od,27/11/2024,2024-11-27T02:52:07Z,"[{""id"":""68wq40z8"",""name"":null,""rel"":""user""}]",qxkowgk8,verified,null,34.38,27/11/2024
